app/templates/displayFilesMacro.html

```diff
@@ -1,21 +1,36 @@
 {% macro displayFiles(filepaths, titleName, deleteLink, databaseId)  %}
-<label class="mb-2"><strong>{{titleName}}</strong></label>
 <table>
+    <tr>
+        <td class="py-2" ><strong>Display</strong></td>
+        <td class="py-2 px-3"><strong>{{ titleName }}</strong></td>
+    </tr>
     {% for key, value in filepaths.items() %}
-        <tr id="attachment_{{value[1]}}" class="attachmentRow_{{key}}" data-id= "{{key}}">
-            <td class="pr-5">
-            {% set filename = key[:8]+ "..." +key[-10:] if key| length > 25 else key  %}
-                <a class="mr-5 fileName" data-filename='{{key}}' href="{{value[0]}}" target="_blank">{{filename}}</a>
+        <tr id="attachment_{{ value[1] }}" class="attachmentRow_{{ key }}" data-id="{{ key }}">
+            <td class="py-2 px-3">
+                {% if value[-1].isDisplayed %}
+                    <input type="checkbox" class="attachmentCheck" data-id="{{ value[1] }}" checked>
+                
+                {% else %}
+                {% if not (value[0].endswith('.png') or value[0].endswith('.jpg') or value[0].endswith('.svg') or value[0].endswith('.jpeg') or value[0].endswith('.gif')) %}
+                        <input type="checkbox" class="attachmentCheck" data-id="{{ value[1] }}" disabled >
```

Comment:
Instead of adding a disabled checkbox, maybe a "-" or just omitting the checkbox altogether would be a better option.

Reply:
I think we should keep them for consistency. what do you think?